### PR TITLE
Request to remove matthias-bs/ESP32_ATC_MiThermometer_Library

### PR DESCRIPTION
**Pull request by matthias-bs**
https://github.com/matthias-bs/ATC_MiThermometer should be used instead

### CODE REVIEW

**Review by per1234 — COMMENTED** [review]
Hello @matthias-bs. I am confirming your request.

Unlike submissions, this type of request is still handled manually. The Library Manager index maintainer occasionally comes through to clear all the outstanding requests. They will merge this pull request when it is done. You will get a notification from GitHub at that time.

**Comment by matthias-bs** [comment]
Thank you!

**Review by MatteoPologruto — APPROVED** [review]

**Comment by MatteoPologruto** [comment]
Done:
```
2023/02/22 17:06:42 Loaded 5572 libraries from DB
Removing ATC MiThermometer Library
Success!
```

**Pull request merged**
